bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_sofie/_parser/_keras/layers/batchnorm.py

```diff
@@ -0,0 +1,48 @@
+from cppyy import gbl as gbl_namespace
```

Comment:
maybe good to check if the datatypes for the input tensors are float since we don't support the other cases?